src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -75,6 +75,7 @@ public function __construct(
 
     /**
      * @Given there is user :email identified by :password
+     * @Given there was account of :email with password :password
```

Comment:
I'm not sure if we don't have a small conflict here. In #4235 I used transformer to transform `:email` into Customer. I'm pretty curious how this sense will be interpreted after merging this two concepts.